src/progress-indicator/radial.scss

```diff
@@ -218,3 +237,8 @@
     }
   }
 }
+
+@mixin iui-progress-indicator-radial-size($size:$iui-icons-xl) {
```

Comment:
Is it worth having default value? This mixin is only used in this file so I would say explicitly pass size in default case.

Reply:
Makes sense, I have removed the default value.